Application/EdFi.Ods.Standard/Artifacts/Metadata/DomainMetadata.xml

```diff
@@ -1,1106 +0,0 @@
-<?xml version="1.0" encoding="utf-8" ?>
```

Comment:
Did you do anything to check to make absolutely sure that this isn't used anywhere and can be removed now?

Reply:
Updated techdocs. Added XML file back in project.